examples/license-classifications.yml

```diff
@@ -3,377 +3,338 @@
 # https://github.com/nexB/scancode-toolkit/commit/ed644e4
 #
 # To demonstrate how one can insert a custom written offer
-# include_source_code_offer_in_notice_file has been set to
+# include-source-code-offer-in-notice-file has been set to
 # true for all licenses of the GPL family.
 categories:
 - name: "copyleft"
 - name: "strong-copyleft"
 - name: "copyleft-limited"
 - name: "permissive"
+  description: "Licenses with permissive obligations."
 - name: "public-domain"
+- name: "include-in-notice-file"
+  description: >
```

Comment:
Please use no-newline-at-end "chomping" style, i.e. use `>-` instead of `>`. Also see https://yaml-multiline.info/.

Reply:
Also below.